docs/getting-started/installing-service-binding.md

```diff
@@ -8,7 +8,17 @@ sidebar_position: 1
 
 ## Operator
 
-_How can To be provided soon_
+- Navigate to the `Operators` -> `OperatorHub` in the OpenShift console
+- Select `Developer Tools` category 
+- Select the `Service Binding Operator`
+
+![Service Binding Operator](https://github.com/redhat-developer/service-binding-operator/blob/master/assets/operator-hub-sbo-screenshot.png)
+
+- Select `beta` channel
+- Select `All namespaces on the cluster` Installation mode
+- Select `Automatic` approval strategy
+- Click on `Install`
+
```

Comment:
How do I know/check that the operator is properly running? 
Should we provide a quick sample the user could play with, just to check things are working out properly?

Reply:
I have provided two ways of checking installation.

[1] Kubectl get `sb`

And then finally,
[2] create a ServiceBinding